yoti_python_sdk/client.py

```diff
@@ -78,16 +78,26 @@ def get_activity_details(self, encrypted_request_token):
         receipt = json.loads(response.text).get("receipt")
 
         encrypted_data = proto.current_user(receipt)
+        encrypted_application_profile = proto.current_application(receipt)
 
         if not encrypted_data:
             return ActivityDetails(receipt)
 
         unwrapped_key = self.__crypto.decrypt_token(receipt["wrapped_receipt_key"])
-        decrypted_data = self.__crypto.decipher(
+
+        decrypted_profile_data = self.__crypto.decipher(
             unwrapped_key, encrypted_data.iv, encrypted_data.cipher_text
         )
-        attribute_list = proto.attribute_list(decrypted_data)
-        return ActivityDetails(receipt, attribute_list)
+        decrypted_application_data = self.__crypto.decipher(
+            unwrapped_key, encrypted_application_profile.iv, encrypted_application_profile.cipher_text
+        )
+
+        user_profile_attribute_list = proto.attribute_list(decrypted_profile_data)
+        application_profile_attribute_list = proto.attribute_list(decrypted_application_data)
+
+        return ActivityDetails(
+            receipt, user_profile_attribute_list, decrypted_application_profile=application_profile_attribute_list
```

Comment:
I prefer taking the approach that we have _all_ or _0_ named arguments, so I'll leave it up to you whether we name the other arguments, or take out the `decrypted_application_profile=`